spec/services/edit_edition_service_spec.rb

```diff
@@ -36,5 +36,65 @@
       expect { EditEditionService.call(live_edition, user) }
         .to raise_error("cannot edit a live edition")
     end
+
+    describe "marks an edition as political" do
+      it "sets system_political to true when the edition is identified as political" do
+        allow(PoliticalEditionIdentifier)
+          .to receive(:new)
+          .with(edition)
+          .and_return(instance_double(PoliticalEditionIdentifier, political?: true))
+
+        expect { EditEditionService.call(edition, user) }
+          .to change { edition.system_political }
+          .to(true)
+      end
+
+      it "sets system_political to false when the edition is not identified as political" do
+        edition = build(:edition, system_political: true)
+
+        allow(PoliticalEditionIdentifier)
+          .to receive(:new)
+          .with(edition)
+          .and_return(instance_double(PoliticalEditionIdentifier, political?: false))
+
+        expect { EditEditionService.call(edition, user) }
+          .to change { edition.system_political }
+          .to(false)
+      end
+    end
+
+    describe "associates the edition with a government" do
```

Comment:
These tests are great, but could they also include a test for what happens if a government doesn't exist for the provided date?

Reply:
Will check.